### DISCUSSION TITLE
Red dollar sign?

### DISCUSSION THREAD

**Discussion by MaximilianPs** [renpy/vscode-language-renpy#370] [Q&A]
Usually the dollar sign is blue and the variables are cyan but in that case is different, can you tell me why?

![image](https://github.com/LuqueDaniel/vscode-language-renpy/assets/23430638/6d08bbd3-497e-4f23-842b-ddb0fa1cd845)

**Comment by duckdoom4** [comment]
Hi, thanks for the bug report. This is likely due to this being placed inside a screen block.

That's a bug and will need to be fixed on our end
